avm/ptn/authorization/pim-role-assignment/tests/e2e/mg.eligible/main.test.bicep

```
@@ -0,0 +1,46 @@
+targetScope = 'managementGroup'
+metadata name = ' PIM Eligible Role Assignments (Management Group scope)'
+metadata description = 'This module deploys a PIM Eligible Role Assignment at a Management Group scope using minimal parameters.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The location to deploy resources to.')
+param resourceLocation string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'pimgmin'
+
+@description('Optional. A token to inject into the name of each resource.')
+param namePrefix string = '#_namePrefix_#'
+
+@description('Required. Principle ID of the user. This value is tenant-specific and must be stored in the CI Key Vault in a secret named \'CI-testUserObjectIdstring\'.')
+@secure()
+param testUserObjectIdstring string = ''
```

```suggestion
@description('Required. Principle ID of the user. This value is tenant-specific and must be stored in the CI Key Vault in a secret named \'CI-testUserObjectId\'.')
@secure()
param testUserObjectId string = ''
```
Not sure if it was me 😄 sorry. Must be changed in the other files too